modules/runner/src/integrationTest/java/org/apache/ignite/internal/ssl/ItSslTest.java

```diff
@@ -250,6 +253,148 @@ void jdbcCanConnectWithSsl() throws SQLException {
         }
     }
 
+    @Nested
+    @DisplayName("Given SSL enabled on the cluster and specific cipher enabled")
+    class ClusterWithSslCustomCipher {
+
+        @Language("JSON")
+        String sslEnabledWithCipherBoostrapConfig = "{\n"
+                + "  network: {\n"
+                + "    ssl : {"
+                + "      enabled: true,\n"
+                + "      ciphers: TLS_AES_256_GCM_SHA384,\n"
+                + "      trustStore: {\n"
+                + "        password: \"" + password + "\","
+                + "        path: \"" + escapeWindowsPath(trustStorePath) + "\""
+                + "      },\n"
+                + "      keyStore: {\n"
+                + "        password: \"" + password + "\","
+                + "        path: \"" + escapeWindowsPath(keyStorePath) + "\""
+                + "      }\n"
+                + "    },\n"
+                + "    port: 3345,\n"
+                + "    portRange: 2,\n"
+                + "    nodeFinder:{\n"
+                + "      netClusterNodes: [ \"localhost:3345\", \"localhost:3346\" ]\n"
+                + "    }\n"
+                + "  },\n"
+                + "  clientConnector.ssl: {\n"
+                + "    enabled: true, "
```

Comment:
I changed the configuration and the cipher in the client connection test - it was passing before because default ciphers suite doesn't include the cipher chosen for the client connection.